server/migrations/sql/20191025170040-create-tags-table.sql

```diff
@@ -1,9 +1,14 @@
 CREATE TABLE tags(
     id SERIAL PRIMARY KEY,
     name TEXT NOT NULL CHECK(name <> ''),
-    type_type_id INT NOT NULL,
-    FOREIGN KEY(type_type_id) REFERENCES tag_types(id)
+    tag_type_id INT NOT NULL,
+    FOREIGN KEY(tag_type_id) REFERENCES tag_types(id)
 );
+
+INSERT INTO tags (name, tag_type_id)
+VALUES 
+ ('official', 1), 
```

Comment:
using `1` here is probably wrong

You probably want to do a CTE to select the correct category, `WITH cat AS (SELECT id FROM tag_type WHERE name="category")` before the INSERT should let you use `cat.id` instead of `1` here

Reply:
@gnarf this suggestion came directly from me... I tested it on my local db :P